CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -0,0 +1,718 @@
+Range: Touch
+    You need more pathogens, but your teammates aren't sharing. Perhaps your 
+enemy might be made to donate some? You grab one of your needles and violently 
+scoop some out of them. This isn't exactly how it worked in med school. Make a 
+melee attack against your enemy. If it hits, you get a contagion reward of 4, if 
+it misses, you get nothing. You get a -2 to miss and deal damage equal to 4 * 
+Strength. Ignore armor. 
+
+==Poison: Thiopental Derivative ==
+Contagion Cost: 1 Contagion
+Duration: 	2 Actions
+Range:		Payload (use in combination with other powers)
+    Truth serum. Principle uses are for non-combat situations. While any 
+character could procure similar substances as a role-playing item for about $60 
+worth of chemistry supplies and a chemistry check of DC 80 a piece, the value of 
+this trick is that it's free, almost unlimited and you don't need to worry about 
+smuggling it anywhere.
+    
+    A biologic target dosed with this substance only gets 3 actions on their 
+turn and must pass a Will Save of DC 90 whenever they are asked a question. If
+they pass, they may lie. If they fail, they must answer with total candor. 
+Targets dosed with the Thiopental Derivative in combat are totally honest about
+commiting to shots, and must succeed on the DC 90 will save or they cannot 
+hunker or take superior cover. The effect lasts for one out-of-combat 
+conversation or 5 turns in combat.
+
+==Transmission Vector ==
+Adrenaline Cost:60 Points
+Needle Cost: 1 Needle
+Duration: 	2 Actions
+Range:		5m radius centered at a point up to 30m from the Doc
+Effect:		You add an aerator to the end of your needle and launch it not at 
+one target in particular, but among the fray hoping to hit multiple targets.
+Targets without gas masks have the poison payload of your choice applied to 
+them.
+
+==Poison:	NF-40 ==
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagion
+Duration: 	two weeks
+Range:		Payload (use in combination with other powers)
+    The biologic agent eats away at the target's flesh and biologic parts at an 
+astonishing rate. The target takes 20 + 2d10 damage per turn for the duration of 
+their affliction. They can be cured by decontamination devices or specially 
+trained medics. Targets that are wearing a gas mask, but have any exposed 
+biology are affected by this even if a deployment method says they wouldn't. 
+Use of biologic weapons are severely frowned upon in most civilized regions of 
+space.
+
+==Poison:	Lethal Toxin ==
+Adrenaline Cost: 60 Points
+Contagion Cost: 1 Contagion
+Duration: 	almost instantaneous
+Range:		Payload (use in combination with other powers)
+    A biologic target afflicted with this poison must succeed on a DC 110 FORT 
+save or the target takes 180 damage that ignores shields and armor and silences 
+the target. If the target succeeds on their save, they take 60 damage instead. A 
+silenced target is prevented from using nanite abilities for 10 turns. (Also 
```

Comment:
"nanite" -> "Nanite"

We might want to change the text to say "active abilities that use Nanites" because, strictly speaking, we never formally tag an ability as a "Nanite Ability". Just a thought.

Reply:
Good clarification.